stdlib/public/core/Builtin.swift

```diff
@@ -259,6 +259,13 @@ public func unsafeDowncast<T : AnyObject>(_ x: AnyObject, to type: T.Type) -> T
   return Builtin.castReference(x)
 }
 
+@_inlineable // FIXME(sil-serialize-all)
+@_transparent
```

Comment:
@_transparent implies @_inlineable.

Reply:
I see. I was following conventions elsewhere in the file. Is there a place where these implied attributes are documents (and ideally regression-tested)?

Reply:
There are Sema tests that all three function kinds have enforced restrictions on what their bodies can reference, and SILGen tests that all three are serialized.

The best way to "document" this behavior would be to emit warnings when redundant attributes are specified. Also, I'd like to eliminate @_transparent completely one day, since it doesn't really serve a purpose.